Description: Define _GNU_SOURCE to use X/Open features
 Define _GNU_SOURCE to get nftw() macros (XOPEN_SOURCE_EXTENDED)
 strptime() (XOPEN_SOURCE), pthread_lock_t (POSIX.1-2001) and others.
Author: Steven Chamberlain <steven@pyro.eu.org>
Bug-Debian: https://bugs.debian.org/820155
Last-Update: 2016-05-29
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/gw/dlr_spool.c
+++ b/gw/dlr_spool.c
@@ -64,6 +64,8 @@
  * Stipe Tolj <stolj at kannel dot org>
  */
 
+#define _GNU_SOURCE
+
 #include <unistd.h>
 #include <sys/types.h>
 #include <sys/stat.h>
--- a/gw/smsc/smsc_http.c
+++ b/gw/smsc/smsc_http.c
@@ -111,6 +111,8 @@
  * Tobias Weber <weber@wapme.de>
  */
 
+#define _GNU_SOURCE
+
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <unistd.h>
--- a/test/test_file_traversal.c
+++ b/test/test_file_traversal.c
@@ -58,6 +58,8 @@
  * test_file_traversal.c - simple file traversal testing
  */
 
+#define _GNU_SOURCE
+
 #include <unistd.h>
 #include <sys/types.h>
 #include <sys/stat.h>
